File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_utlmntls.java
// EXPECTED: 72
// Reference emit for UTLMNTLS — UTLMNT00 LINE SEQ control sum.
import java.util.List;
import java.util.Map;
public final class UtlmntlsRef {
  public static void main(String[] args) {
    var funcs = List.of("ARCHIVE", "CLEANUP", "REORG");
    var codes = Map.of("ARCHIVE", 14, "CLEANUP", 24, "REORG", 34, "ANALYZE", 44);
    int total = 0;
    for (String f : funcs) total += codes.get(f);
    System.out.printf("%02d%n", total);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_utlmntls.py
"""Reference emit for UTLMNTLS — UTLMNT00 LINE SEQ control ARCHIVE+CLEANUP+REORG → 72."""
# EXPECTED: 72
FUNCS = ["ARCHIVE", "CLEANUP", "REORG"]
CODES = {"ARCHIVE": 14, "CLEANUP": 24, "REORG": 34, "ANALYZE": 44}
total = sum(CODES[f] for f in FUNCS)
print(f"{total:02d}")
